README.md

```diff
@@ -31,33 +33,50 @@ make
 make check
 sudo make install
 ```
-This may require the installation of additional dependencies.
-Then you will want the golang code generator
+Though this may require the installation of additional dependencies for the build
+process.
+
+Then you will need the golang code generator
 ```
 go get -u github.com/golang/protobuf/protoc-gen-go
 ```
 
-You can run `make install` or `go install` directly.
+##### Installation
+
+To use this tool, install it from code with `make install`, `go install` directly,
+or `go get github.com/infobloxopen/protoc-gen-gorm`.
+
+### Usage
+
+Once installed, the `--gorm-out=.` or `--gorm-out=${GOPATH}src`
+option can be specified in a protoc command to generate the .pb.gorm.go files.
+
+Any types with the `option (orm.opts).ormable = true` will have the following
```

Comment:
Change `orm` to `gorm`, since this is really a gORM-specific plugin.

Reply:
Should ormable -> gormable, too?

Reply:
if the overall "orm" is changed to "gorm" then no, that's not needed.